src/Orc.Controls/Orc.Controls.Shared/Controls/DateTimePicker/Views/DateTimePickerControl.xaml.cs

```diff
@@ -99,6 +128,26 @@ public bool IsReadOnly
         public static readonly DependencyProperty IsReadOnlyProperty = DependencyProperty.Register("IsReadOnly", typeof(bool),
             typeof(DateTimePickerControl), new PropertyMetadata(false));
 
+        public string Format
+        {
+            get { return (string)GetValue(FormatProperty); }
+            set { SetValue(FormatProperty, value); }
+        }
+
+        public static readonly DependencyProperty FormatProperty = DependencyProperty.Register("Format", typeof(string),
+            typeof(DateTimePickerControl), new FrameworkPropertyMetadata(CultureInfo.CurrentCulture.DateTimeFormat.ShortDatePattern + " " + CultureInfo.CurrentCulture.DateTimeFormat.LongTimePattern, (sender, e) => ((DateTimePickerControl)sender).OnFormatChanged()));
+
+        public bool IsHour12Format
+        {
+            get { return (bool)GetValue(IsHour12FormatProperty); }
+            private set { SetValue(IsHour12FormatKey, value); }
+        }
+
+        private static readonly DependencyPropertyKey IsHour12FormatKey = DependencyProperty.RegisterReadOnly("IsHour12Format", typeof(bool),
+            typeof(DateTimePickerControl), new PropertyMetadata(false));
```

Comment:
Because type is string. By default we use format from currentculture, but user can set to custom format (not related to any culture).